src/basic/INDEXLIST.hpp

```diff
@@ -60,6 +60,10 @@ class INDEXLIST : public KernelBase
   void runCudaVariant(VariantID vid, size_t tune_idx);
   void runHipVariant(VariantID vid, size_t tune_idx);
   void runOpenMPTargetVariant(VariantID vid, size_t tune_idx);
+  void runKokkosVariant(VariantID vid, size_t)
+  {
+     getCout() << "\n INDEXLIST : Unknown Kokkos variant id = " << vid << std::endl;
```

Comment:
Does it make more sense for these messages to say "unimplemented" or "unsupported" rather than "unknown"?

Reply:
Sure, I can change the wording to use "unimplemented."